Makefile

```diff
@@ -6,29 +6,49 @@ K8S_VERSION_MINOR := $(word 1,${K8S_VERSION_PARTS}).$(word 2,${K8S_VERSION_PARTS
 
 MAKEFILE_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
+# expands to 'true' if PACKER_VARIABLE_FILE is non-empty
+# and the file contains the string passed as the first argument
+# otherwise, expands to 'false'
+packer_variable_file_contains = $(if $(PACKER_VARIABLE_FILE),$(shell grep -Fq $1 $(PACKER_VARIABLE_FILE) && echo true || echo false),false)
+
+vercmp = $(shell $(MAKEFILE_DIR)/files/bin/vercmp "$1" "$2" "$3")
+
+in_iso_region = $(if $(findstring us-iso,$(aws_region)),true,false)
+
+# gp3 volumes are used by default for 1.26+
+# TODO: remove this when 1.25 reaches EOL
+ifeq ($(call packer_variable_file_contains,volume_type), false)
+	ifeq ($(call vercmp,$(kubernetes_version),gteq,1.26.0), true)
```

Comment:
This going to actually happen for 1.26?

Reply:
Did you forget to push a commit again?

Reply:
I was updating things as I responded, it's pushed now 👍